odoo/http.py

```diff
@@ -696,6 +716,11 @@ def route(route=None, **routing):
     :param bool csrf: Whether CSRF protection should be enabled for the
         route. Enabled by default for ``'http'``-type requests, disabled
         by default for ``'json'``-type requests.
+    :param bool npdb: Specifies that this route should always go through
+        the nodb routing (serve_nodb)
+    :param bool readonly: Whether this endpoint should use open a cursor
```

Comment:
the readonly param can also be a callable(registry, request) -> bool, it must be documented